maestro/financial-connections/Testmode-Data-TestOauthInstitution.yaml

```diff
@@ -4,13 +4,10 @@ appId: com.stripe.android.financialconnections.example
 - startRecording: /tmp/test_results/testmode-data-testoauthinstitution
 # Android specific: Navigate to example
 - tapOn: "Playground"
-- tapOn:
-    id: "Native_checkbox"
-- tapOn: "Mode_Dropdown_Icon"
-- tapOn:
-    id: "Test_checkbox"
-- tapOn:
-    id: "Data_checkbox"
+- tapOn: "Merchant"
+- tapOn: "Test"
+- tapOn: "Native"
+- tapOn: "Data"
```

Comment:
Updated e2e tests to reference texts.

(we'll be triggering e2e tests via deep link on a following PR anyways, this is just a temporary change)